src/Type/ListType/AbstractListType.php

```diff
@@ -77,7 +77,7 @@ public function getTypeOf()
 
     public function parseValue($value)
     {
-        foreach ($value as $keyValue => $valueItem) {
+        foreach ((array) $value as $keyValue => $valueItem) {
```

Comment:
Should it be null or an empty array? I remember we talked about it but not sure what have we ended up with as the default value..